src/app/frontend/deploy/createsecret.html

```diff
@@ -0,0 +1,64 @@
+<!--
+Copyright 2015 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<md-dialog aria-label="Create a new secret" layout="column" layout-padding>
+  <md-content layout-padding>
+    <h4 class="md-title">Create a new image pull secret</h4>
+    <p>The new secret will be added to the cluster.</p>
```

Comment:
In line 36 you use wording `will be created` and here `will be added`. Please unify.